### PR TITLE
fix/aut-657 Reorder to min number

### PR DESCRIPTION
**Pull request by meefox**
Related to https://oat-sa.atlassian.net/browse/AUT-657

**Description:**
For GraphicOrderInteraction changed a starting order from the minimum available number, 
Bonus: fixed an issue when clicking on text label was not register an event and throw an error to console.

**Changes:**
* Changed active when unselecting shape.
* Add pointer-events none to label and remove unused code
* Added missed UT 

**How to check:**
* Create an item with GraphicOrderInteraction
* Add multiple shapes (5 is enough)
* Go to Responses 
* Select in order 1,2,3,4,5
* Deselect in order 1,5,2 
* See selection is 1 all the time (or minimum available)
* An extra check, clicking on the number itself will not throw an error anymore

**Requires:**
* Swap in package.json to `"@oat-sa/tao-item-runner-qti": "github:oat-sa/tao-item-runner-qti-fe#fix/AUT-657_reorder-to-min-number"`(fixed) for taoQTIItem extention.
* Rebundle `npx grunt taoqtiitembundle` from tao/views/build.

### CODE REVIEW

**Review by jsconan — COMMENTED** [review]
It looks better now, but another issue was added, when you click the text.

File: src/qtiCommonRenderer/renderers/interactions/GraphicOrderInteraction.js [hunk 1]
```diff
@@ -220,22 +233,10 @@ var _createTexts = function _createTexts(paper, size) {
...
-                if (element.data('number') === number && element.events) {
-                    //we just need to retrieve the right element
-                    //call the click event
-                    var evt = _.where(element.events, { name: 'click' });
-                    if (evt.length && evt[0] && typeof evt[0].f === 'function') {
-                        evt[0].f.call(element);
-                    }
-                }
-            });
-        });
```
[the remark this answers is not in the document]
Comment (jsconan) [reply src/qtiCommonRenderer/renderers/interactions/GraphicOrderInteraction.js:238]: Nevermind, after I refreshed my environment and reinstalled the patch I was not able to reproduce this.

**Review by jsconan — APPROVED** [review]
It works well.

A side note about the description in which the given link for node.js contains a typo...
It should rather be `"@oat-sa/tao-item-runner-qti": "github:oat-sa/tao-item-runner-qti-fe#fix/AUT-657_reorder-to-min-number"`

And last but not least, regarding the unit test, it is nice to have it but the sample image would have been better with a smaller weight. It does not need to be beautiful. A nasty jpeg with plain color would have been enough. Anyway, the file is now part of the history, so its weight will already make the repo heavier even if removed now...

**Review Checklist**
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] ~~New code is not subject to concurrency issues (if applicable)~~
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Comment by AntonNeviarouski** [comment]
QA verified

**Pull request merged**